harvard-university-of-worcester.csl

```diff
@@ -34,7 +34,7 @@
   </macro>
   <macro name="author-short">
     <names variable="author">
-      <name form="short" and="text" delimiter-precedes-last="never"/>
+      <name form="short" delimiter=" " and="text" delimiter-precedes-last="never" sort-separator=" "/>
```

Comment:
Neither of the two new attributes (delimiter and sort-separator) you introduce here will ever be used. Delete them.